pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -795,11 +801,16 @@ metadata:
   name: jiva-volume-create-patchreplicadeployment-default
```

Comment:
it seems - this runtaks is not in use.. we can think of removing it

Reply:
removed